config.yaml

```diff
@@ -8,7 +8,7 @@ global:
     EUROCXOF: '0xed35e46b095197da30ddffa5b91d386886d5ce0d'
     KESUSD: '0xbAcEE37d31b9f022Ef5d232B9fD53F05a531c169'
     USDTUSD: '0xE06C10C63377cD098b589c0b90314bFb55751558'
-
```

Comment:
CELOPHP rateFeedID is missing here

Reply:
@philbow61 You specified to remove it

Reply:
@RyRy79261  Me and Philip II created a new metric that does make sense with the chainlink setup.
It's an isOldestReportExpired() call to sortedOracles. that's why we need to add that rateFeed again. 
You can see that we are also referencing it in the definition of that isOldestReportExpired() metric.

Reply:
i re-added it, got lost in the back and forth